### ISSUE TITLE
[feature request] Automatic Downloads system  "schedule download" 

### ISSUE THREAD

**Issue by TheLorry** [inorichi/tachiyomi-extensions#3782]
this is probably a low priority addition, But anyway, I think that a Scheduled Downloads function would be useful. Something like the current backup system, but with focuos in download the recently released chapters of each mangá we want. 

USER BENEFITS
The certainty that they will always be up to date with the work that we follow, regardless of whether we are online or not.
A more and precise control of the manga we download.
And the convenience of programming what the Downloads will be like only once.

REQUIREMENTS
I thought about having a "Download Scheduled button" On the page with the synopsis of the manga, a button that would open a sub-menu with the Download options (which are currently in the chapters tab) and "Automatic download of new chapters" option.

![45 Sem Título_20200713035748](https://user-images.githubusercontent.com/56932297/87330954-ab756400-c50f-11ea-990a-7e35ed3691f2.png)

And a second button on the current Downloads tab that would take you to a list of the manga / novels that would be programmed for Automatic Download.defining a periodic frequency

![45 Sem Título_20200713035715](https://user-images.githubusercontent.com/56932297/87329913-263d7f80-c50e-11ea-8bd7-a1951afbda83.png)

**Comment by CarlosEsco** [comment]
But there already is auto download?

**Comment by SnakeDoc83** [comment]
Also, not something that would be done at the extension level.

**Comment by TheLorry** [comment]
my description must have been confused, but the general idea, was that the user could define a Manga to download new chapters soon as it was available. Doing something like a weekly scan in the sources looking for new chapters of certain manga. I think the app's notification system already does that, so the idea was to have something similar but downloading the new chapters, Not just notified that they are available.

**Comment by arkon** [comment]
Settings > Downloads > Download new chapters
